package.json

```diff
@@ -12,6 +12,7 @@
     "types/*.d.ts"
   ],
   "scripts": {
+    "prepublish": "npm run build",
```

Comment:
hm .. so instead of typing `npm run build`, people will type `npm run prepublish` ? 🤔